src/invidious/views/confirm_leave.ecr

```diff
@@ -0,0 +1,22 @@
+<% content_for "header" do %>
+<title><%= translate(locale, "Leave") %> - Invidious</title>
+<% end %>
+
+<div class="h-box">
+    <h3><%= translate(locale, "You are leaving Invidious. Continue to external link?") %></h3>
```

Comment:
You need to add this line to `locales/en-US.json`, otherwise Weblate won't be able to detect it, and in turn that won't be proposed to translators:

```json
"confirm_dialog_external_link": "You are leaving Invidious. Continue to external link?"
```

```suggestion
    <h3><%= translate(locale, "confirm_dialog_external_link") %></h3>
```